index.js

```diff
@@ -3,126 +3,223 @@ const express = require('express'),
   morgan = require('morgan'),
   bodyParser = require('body-parser'),
   uuid = require('uuid');
+  const mongoose = require('mongoose');
+  const Models = require('./models.js');
 
 const app = express();
+const Movies = Models.Movie;
+const Users = Models.User;
+mongoose.connect('mongodb://localhost:27017/myFlixDatabase', {
+  useNewUrlParser: true, // Connect the DB (to perform CRUD)
+  useUnifiedTopology: true
+});
 
 app.use(morgan('common'));
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true}));
 
 // GET requests
 app.get('/', (req, res) => {
   res.send('Welcome to myFlix!');
 });
 
-// CREATE a new User
-app.post('/users', (req, res) => {
-  const newUser = req.body;
-
-  if (newUser.name) {
-    newUser.id = uuid.v4();
-    users.push(newUser);
-    res.status(201).json(newUser)
-  } else {
-    res.status(400).send('New user needs a name.')
-  }
+// Read/ Return a list of all movies
+app.get('/movies', (req, res) => {
+  Movies.find()
+    .then((movies) => {
+      res.status(201).json(movies);
```

Comment:
This is a get endpoint, we should use status code of `200` meaning success not `201` for created.